src/main/java/mate/academy/Main.java

```diff
@@ -2,16 +2,26 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
+import mate.academy.service.OrderService;
 
 public class Main {
     public static void main(String[] args) {
-        MovieService movieService = null;
+        Injector injector = Injector.getInstance("mate.academy");
+        final MovieService movieService = (MovieService) injector
+                .getInstance(MovieService.class);
+        final CinemaHallService cinemaHallService = (CinemaHallService) injector
+                .getInstance(CinemaHallService.class);
```

Comment:
Syntax Error: Missing semicolon at the end of the statement. Please add a semicolon after `getInstance(CinemaHallService.class)`.